src/postamble_minimal.js

```diff
@@ -218,8 +218,13 @@ WebAssembly.instantiate(Module['wasm'], imports).then(/** @suppress {missingProp
 
   initRuntime(wasmExports);
 
-#if PTHREADS
-  PThread.loadWasmModuleToAllWorkers(ready);
+#if PTHREADS && PTHREAD_POOL_SIZE
+  var workersReady = PThread.loadWasmModuleToAllWorkers();
+#if PTHREAD_POOL_DELAY_LOAD
+  ready();
+#else
+  workersReady.then(ready);
```

Comment:
Why do we need both the `await` on line 117, and this `then`?

Reply:
The `await` is what the non-`MINIMAL_RUNTIME` does, this code here is what `MINIMAL_RUNTIME` does instead.

Reply:
See the comment on line 111-112 of libpthread.js

Reply:
Oh, my brain somehow missed "minimal" in the name of the file here... thanks